[ERROR] AttributeError: 'TraderConfig' object has no attribute 'stoploss_strategy_name'

```bash
[ERROR] AttributeError: 'TraderConfig' object has no attribute 'stoploss_strategy_name'
Traceback (most recent call last):
  File "/var/task/main.py", line 21, in lambda_handler
    trader.apply_trading_rule()
  File "/var/task/models/real_trader.py", line 51, in apply_trading_rule
    self.__play_scalping_trade(candles)
  File "/var/task/models/real_trader.py", line 143, in __play_scalping_trade
    new_stop: float = self.__drive_trail_process(candles.iloc[-2], last_indicators)
  File "/var/task/models/real_trader.py", line 182, in __drive_trail_process
    possible_stoploss: float = self.stoploss_method()(
  File "/var/task/models/real_trader.py", line 40, in stoploss_method
    return stoploss_strategy.STRATEGIES[self.config.stoploss_strategy_name]
```

--- a/models/trader_config.py
+++ b/models/trader_config.py
@@ -118,3 +118,7 @@ class TraderConfig:
     @property
     def stoploss_buffer_base(self) -> float:
         return self._entry_rules['stoploss_buffer_base']
+
+    @property
+    def stoploss_strategy_name(self) -> str:
+        return self._stoploss_strategy_name

--- a/tests/test_trader_config.py
+++ b/tests/test_trader_config.py
@@ -1,3 +1,4 @@
+import os
 from typing import Dict, List, Union
 from unittest.mock import patch
 import pytest
@@ -112,3 +113,13 @@ class TestGetStoplossBufferBase:
 
     def test_basic(self, config: TraderConfig, selected_entry_rules: Dict[str, Union[int, float]]):
         assert config.stoploss_buffer_base == selected_entry_rules['stoploss_buffer_base']
+
+
+class TestGetStoplossStrategyName:
+    @pytest.fixture(name='set_stoploss_strategy', scope='class')
+    def fixture_set_stoploss_strategy(self):
+        os.environ['STOPLOSS_STRATEGY'] = 'hoge'
+        yield
+
+    def test_basic(self, config: TraderConfig):
+        assert config.stoploss_strategy_name == os.environ.get('STOPLOSS_STRATEGY')

